### PR TITLE
[simpleini] update to v4.1.9

### LINKED ISSUE #23804: [ simpleini ] update to v4.1.9
Library name: [simpleini](https://github.com/brofield/simpleini)

New version number: [v4.1.9](https://github.com/brofield/simpleini/releases/tag/v4.19)

Other information that may be useful (release notes, etc...)

The recent 4.1.8 is a [rollup release](https://github.com/brofield/simpleini/releases/tag/4.18) for all the commits in the last 9 years for the library. Author did not provide a release note.

v4.1.9 has release note of "Bug fix release and roll-up of recent changes".

### PR DESCRIPTION
**Pull request by FrankXie05**
Fix #23804 

Update [simpleini] update to v4.1.9.

No features need to test.

### CODE REVIEW

**Review by JonLiu1993 — APPROVED** [review]

File: ports/simpleini/portfile.cmake [hunk 1]
```diff
@@ -13,8 +13,7 @@ set(SIMPLEINI_SOURCE ${SOURCE_PATH}/SimpleIni.h
                      ${SOURCE_PATH}/ConvertUTF.c
 )
 
-file(INSTALL ${SIMPLEINI_SOURCE} DESTINATION ${CURRENT_PACKAGES_DIR}/include)
-# Install sample
-file(INSTALL ${SOURCE_PATH}/snippets.cpp DESTINATION ${CURRENT_PACKAGES_DIR}/share/sample)
+set(VCPKG_POLICY_EMPTY_INCLUDE_FOLDER enabled)
```
Comment (strega-nil-ms) [inline ports/simpleini/portfile.cmake:16]: This makes no sense. It's a header-only library; this is suddenly not installing anything. Why did you change the old version when it should still work?

**Review by strega-nil-ms — CHANGES_REQUESTED** [review]

File: ports/simpleini/portfile.cmake (hunk 1 above)
Comment (FrankXie05) [reply ports/simpleini/portfile.cmake:16]: My fault, it still install, I'll fix it right away. :)

File: ports/simpleini/portfile.cmake [hunk 2]
```diff
@@ -14,7 +14,6 @@ set(SIMPLEINI_SOURCE ${SOURCE_PATH}/SimpleIni.h
 )
 
 file(INSTALL ${SIMPLEINI_SOURCE} DESTINATION ${CURRENT_PACKAGES_DIR}/include)
-# Install sample
-file(INSTALL ${SOURCE_PATH}/snippets.cpp DESTINATION ${CURRENT_PACKAGES_DIR}/share/sample)
```
Comment (FrankXie05) [inline ports/simpleini/portfile.cmake:18]: In the current version, this file has been removed

**Review by JonLiu1993 — APPROVED** [review]

**Comment by strega-nil-ms** [comment]
Thanks!
